uast/transformer/ops.go

```diff
@@ -853,7 +866,11 @@ func (o Fields) ConstructObj(st *State, obj nodes.Object) (nodes.Object, error)
 				continue
 			}
 		}
-		v, err := f.Op.Construct(st, nil)
+		op := f.Op
+		if f.Drop && op == nil {
```

Comment:
So the comment above should be "avoid using drop with an op", shouldn't it? Maybe `f.Drop && op != nil` should be an error in this case.

Reply:
No, it's actually a preferred way of doing it. It reads like this: "drop if the field not exists, or if it matches the Op".

Reply:
I just realised that `Drop` with an op could be used to restrict the kind of values dropped instead of always dropping the field (for example, 'drop this field if the string value is "foo"') so I said nothing :)